lib/plugins/aws/package/compile/events/msk/index.js

```diff
@@ -40,6 +40,79 @@ class AwsCompileMSKEvents {
       additionalProperties: false,
       required: ['arn', 'topic'],
     });
+
+    const secretsManagerArnRegex =
+      'arn:[a-z-]+:secretsmanager:[a-z0-9-]+:\\d+:secret:[A-Za-z0-9/_+=.@-]+';
+
+    this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'kafka', {
```

Comment:
I believe in this case it would be better to go with separate module/class responsible for `kafka` event compilation, as in current form, not much is shared between the two and both filename and class here suggest that it's only responsible for `msk` event. What do you think? Do you think there's a reason to keep them both in this module?

Reply:
Yeah I think that's reasonable. I think at first I kept them in the same file since they were mostly related. But I think it makes sense to separate them.